integration-cli/docker_cli_login_test.go

```diff
@@ -14,8 +15,7 @@ func (s *DockerSuite) TestLoginWithoutTTY(c *check.C) {
 	cmd.Stdin = bytes.NewBufferString("buffer test string \n")
 
 	// run the command and block until it's done
-	if err := cmd.Run(); err == nil {
-		c.Fatal("Expected non nil err when loginning in & TTY not available")
-	}
+	err := cmd.Run()
+	c.Assert(err, checker.NotNil, check.Commentf("Expected non nil err when loginning in & TTY not available"))
```

Comment:
@MHBauer not sure of what you mean ?_?. the `check.Commentf(…)` will be printed in case of failure of the test. It adds some context in case of failure. Putting into a `golang` comment won't add too much context in case of failure :stuck_out_tongue_closed_eyes:.